lms/djangoapps/instructor/enrollment_report.py

```diff
@@ -0,0 +1,47 @@
+"""
+Defines abstract class for the Enrollment Reports.
+"""
+from instructor.enrollment_report_provider import EnrollmentReportProvider
+from django.contrib.auth.models import User
+import collections
+
+
+class BaseEnrollmentReportProvider(EnrollmentReportProvider):
+    """
+    The base abstract class for all Enrollment Reports that can support multiple
+    backend such as MySQL/Django-ORM.
+
+    # don't allow instantiation of this class, it must be subclassed
+    """
+    def get_user_profile(self, user_id):
+        """
+        Returns the UserProfile information.
+        """
+        user_info = User.objects.select_related('profile').get(id=user_id)
+        user_data = collections.OrderedDict()
+        user_data['User ID'] = user_info.id
+        user_data['Username'] = user_info.username
+        user_data['First Name'] = user_info.first_name
+        user_data['Last Name'] = user_info.last_name
+        user_data['Language'] = user_info.profile.language
+        user_data['Location'] = user_info.profile.location
+        user_data['Year of Birth'] = user_info.profile.year_of_birth
+        user_data['Gender'] = user_info.profile.gender
+        user_data['Level of Education'] = user_info.profile.level_of_education
+        user_data['Mailing Address'] = user_info.profile.mailing_address
+        user_data['Goals'] = user_info.profile.goals
+        user_data['City'] = user_info.profile.city
+        user_data['Country'] = user_info.profile.country
+        return user_data
```

Comment:
THis isn't quite all the data we need. Here are the columns that we are expecting (from the JIRA):

- get_user_profile(user_id): returns  username, fullname, and email, language, location, year of birth, gender, level of education, mailing_address, goals for joining site, first-name, last-name, state, country, Company name, title

I think we're just missing: fullname, company name, title. I believe company name/title are stored as a JSON string in profile.meta. Let me know if you can't find that info.